config/marconi.mk

```diff
@@ -5,7 +5,7 @@ INCLUDE += -I$(NETCDF_INC) -I$(HDF5_INC)
 GLFLAGS  = -lm
 CC=icc
 MPICC=mpiicc
-OPT=-O3 -xHost
+OPT=-O3 -xHost -fma -xMIC-AVX512 -finline-functions -align
```

Comment:
If it is specific for the MIC architecture (from wikipedia I get that KNL is a kind of MIC architecture, am I right here?) a suggestion could be to add it to the mic section in the file feltor/config/devices/devices.mk 
Then programs could be compiled with e.g. 
make arakawa_mpib device=mic
when we are on the A2 partition and
make arakawa_mpib device=omp
when we are on the A1 partition
What do you think?